webapp/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "webapp",
-  "version": "1.26.0",
+  "version": "1.27.1",
```

Comment:
As we didn't released anything yet, we can have this as 1.27.0 instead of bumping the version during the PR time

Reply:
It's complicated because when we create a new commit after changing the package.json, it becomes difficult to track the deployment version. So I always prefer creating a new version in the package.json for the commits.